app/controllers/application_controller.rb

```diff
@@ -64,9 +65,17 @@ def check_for_read_only_mode
     true
   end
 
+  def remove_non_lobster_cookies
```

Comment:
🤔 Not sure about this name. Open to suggestions.

Reply:
```suggestion
  def remove_unknown_cookies
```

How's this?